### PR TITLE
feat: add target and max blob per electra and peerdas

### PR DESCRIPTION
**Pull request by barnabasbusa**

### CODE REVIEW

File: config-example/cl/config.yaml [hunk 1]
```diff
@@ -132,7 +132,23 @@ MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS: 4096
...
+# 2**8 * 10**9 (= 256,000,000,000)
+MAX_PER_EPOCH_ACTIVATION_EXIT_CHURN_LIMIT: 256000000000
+# `9`
+BLOB_SIDECAR_SUBNET_COUNT_ELECTRA: $MAX_BLOBS_PER_BLOCK_ELECTRA
+# `uint64(6)`
+TARGET_BLOBS_PER_BLOCK_ELECTRA: $TARGET_BLOBS_PER_BLOCK_ELECTRA
+# `uint64(9)`
+MAX_BLOBS_PER_BLOCK_ELECTRA: $MAX_BLOBS_PER_BLOCK_ELECTRA
+# MAX_REQUEST_BLOCKS_DENEB * MAX_BLOBS_PER_BLOCK_ELECTRA
+MAX_REQUEST_BLOB_SIDECARS_ELECTRA: $(($MAX_REQUEST_BLOCKS_DENEB * $MAX_BLOBS_PER_BLOCK_ELECTRA))
```
Comment (pk910) [inline config-example/cl/config.yaml:151]: that syntax won't work as we're using that weird envsubst.py hack for this file too?
https://github.com/ethpandaops/ethereum-genesis-generator/blob/master/entrypoint.sh#L81

Comment (barnabasbusa) [reply config-example/cl/config.yaml:151]: it is indeed not working :D maybe some bash magic with your solution will work?

Comment (barnabasbusa) [reply config-example/cl/config.yaml]: Should be fixed now.

File: defaults/defaults.env [hunk 2]
```diff
@@ -40,6 +40,12 @@ export SHARD_COMMITTEE_PERIOD="${SHARD_COMMITTEE_PERIOD:-256}"
...
 export DATA_COLUMN_SIDECAR_SUBNET_COUNT="${DATA_COLUMN_SIDECAR_SUBNET_COUNT:-128}"
-export MAX_BLOBS_PER_BLOCK="${MAX_BLOBS_PER_BLOCK:-6}"
+export MAX_BLOBS_PER_BLOCK_ELECTRA="${MAX_BLOBS_PER_BLOCK_ELECTRA:-9}"
+export TARGET_BLOBS_PER_BLOCK_ELECTRA="${TARGET_BLOBS_PER_BLOCK_ELECTRA:-6}"
+export MAX_BLOBS_PER_BLOCK_EIP7594="${MAX_BLOBS_PER_BLOCK_EIP7594:-12}"
+export TARGET_BLOBS_PER_BLOCK_EIP7594="${TARGET_BLOBS_PER_BLOCK_EIP7594:-9}"
 export EL_PREMINE_ADDRS="${EL_PREMINE_ADDRS:-{}}"
+export MAX_REQUEST_BLOCKS_DENEB="${MAX_REQUEST_BLOCKS_DENEB:-128}"
+export MAX_REQUEST_BLOB_SIDECARS_ELECTRA=$(($MAX_REQUEST_BLOCKS_DENEB * $MAX_BLOBS_PER_BLOCK_ELECTRA))
+export MAX_REQUEST_BLOB_SIDECARS_EIP7594=$(($MAX_REQUEST_BLOCKS_DENEB * $MAX_BLOBS_PER_BLOCK_EIP7594))
```
Comment (pk910) [inline defaults/defaults.env:50]: nice idea,  but we can't do this in here.
the defaults are loaded before any user defined value gets applied:
https://github.com/ethpandaops/ethereum-genesis-generator/blob/master/entrypoint.sh#L4

so the expressions will be evaluated with default values only.

this needs to happen somewhere in the entrypoint script, after all values are available :)

**Pull request merged**